test/models/partner_test.rb

```diff
@@ -4,107 +4,99 @@
 
 class PartnerTest < ActiveSupport::TestCase
   setup do
-    @partner = create(:partner)
```

Comment:
Tiny nitpick: Is there any reason to use `FactoryBot.{build,create}`, when you can just do `build()` or `create()`? Most tests do the latter